Update activity-reports.md (#12295)
MicrosoftDocs/microsoft-365-docs: microsoft-365/admin/activity-reports/activity-reports.md

You can easily see how people in your business are using Microsoft 365 services. For example, you can identify who is using a service a lot and reaching quotas, or who may not need a Microsoft 365 license at all. Perpetual license model won't be included in the reports. 
  

> Reports are available for the last 7 days, 30 days, 90 days, and 180 days. Data won't exist for all reporting periods right away. The reports typically become available within 48 hours, but sometimes take several days
> before becoming available.

Reviewer: ```suggestion
Reports are available for the last 7 days, 30 days, 90 days, and 180 days. Data won't exist for all reporting periods right away. The reports typically become available within 48 hours, but might sometimes take several days
to become available.
```
